.github/workflows/default_on_push.yml

```diff
@@ -46,8 +46,8 @@ jobs:
           context: .
           file: ./Dockerfile
           build-args: |
-            VCS_REF=${{ github.sha }}
-            VCS_VER=${{ github.ref }}
+            VCS_REVISION=${{ github.sha }}
+            VCS_VERSION=${{ github.ref }}
```

Comment:
In reference to [associated commit message](https://github.com/docker-mailserver/docker-mailserver/pull/2752/commits/05c533e8dd0a065367c1b995522be2d4fd7a128a):

I cannot track down the PR (perhaps it was a direct commit) that introduced this for more information.

Only where it was introduced before these workflow files with just [`Makefile` usage](https://github.com/docker-mailserver/docker-mailserver/pull/1256#issuecomment-529406724). It looks like whomever added this to the workflows misunderstood it?

I don't see why `VCS_VERSION` is assigned `github.ref`, but maybe my memory is bad and that will use a tag when it's a tag event. Otherwise AFAIK it would just be the branch ref it's running from.

It's intention was to just bake in the info into image metadata so it could be queried and look like:

```
org.label-schema.vcs-ref=7580aec
org.label-schema.version=v6.1.0-111-g7580aec
```

---

`VCS_REF` probably referred to [`git ref`](https://git-scm.com/book/en/v2/Git-Internals-Git-References). That's the commit SHA for the inverse of `github.ref` that `VCS_VER` has been using here (_while the `Makefile` instead uses [`git describe`](https://git-scm.com/docs/git-describe), so isn't consistent_).